### DISCUSSION TITLE
Resistant Damage Interacting with Attacks and Body Damage

### DISCUSSION THREAD

**Discussion by GooseThatCould** [dmdorman/hero6e-foundryvtt#115] [Q&A]
Hi,
So quick question, I was wondering if someone could double check this for me, but when I attempt to do an attack, the body damage dealt is not what it should be based on the damage of the attack or the resistant defence of the target. This might just be me being dumb and having a fundamental misunderstanding of the system, and if so, please ignore and forgive me. Otherwise, let me know if you can replicate the result of it being an incorrect calculation. 
Thanks!

**Comment by aeauseth** [comment]
Please attach screenshots of the attack, damage, and apply damage cards from the chat window.  There are differences in how resistant defenses work based on a normal vs killing attack.

**Reply by GooseThatCould** [reply]
You are correct, what was confusing me was the minimum damage rule, which I didn't know about before. I apologize for this, I'll keep a look out for things, but I'll make sure to clarify my knowledge of the rules before stating anything here. Sorry! Keep up the fantastic work, Hero System is really a lot to deal with, so any automating assistance is super helpful.

**Comment by aeauseth** [comment]
Glad I could point you in the right direction.
